packages/docusaurus-utils/src/gitUtils.ts

```diff
@@ -39,6 +43,18 @@ export const getFileCommitDate = (
   const fileBasename = path.basename(file);
   const fileDirname = path.dirname(file);
 
+  if (
+    shell.exec(`git ls-files --error-unmatch -- "${fileBasename}"`, {
+      // cwd is important, see: https://github.com/facebook/docusaurus/pull/5048
+      cwd: fileDirname,
+      silent: true,
+    }).code !== 0
```

Comment:
Is it necessary to execute another command? I think an empty history is enough to tell that the file is not tracked?

Reply:
I understand. Well, I cannot claim that it's always the case when the output is blank (although I will do some research about it tomorrow).

In the worst case, we can catch and print the error in a more generic way, like:

Failed to fetch update time for one or more files

Reply:
I think we can do it.